core/src/test/codegen/config.fmpp

```diff
@@ -390,11 +391,16 @@ data: {
     ]
 
     # List of methods for parsing extensions to "CREATE [OR REPLACE]" calls.
-    # Each must accept arguments "(SqlParserPos pos, boolean replace)".
+    # Each must accept arguments "(SqlParserPos pos, SqlCreateOrReplace replace)".
```

Comment:
We should change all of the occurrences of "replace" with either "createType" or "createOption" (depending on what your preference is based on my comment on the enum name).

The same is true for all of the locations you've updated throughout the code.

Reply:
Agreed, just changed all "replace" params to createSpecifier.